buddyboss/buddyboss-platform
# Fixed - Private message threads does not loads more then 20

### All Submissions:


Fixes #2051 

### How to test the changes in this Pull Request:

1. Go To Message screen
2. Try to load more thread by scrolling down
3. It should load more threads now

### Proof Screenshots or Video

https://www.loom.com/share/52fc4779bfa84713b02fbb57ffacb917

### Other information:
